addons/lunch/static/src/js/lunch_widget.js

```diff
@@ -60,20 +59,12 @@ var LunchWidget = Widget.extend({
         var self = this;
         var superDef = this._super.apply(this, arguments);
 
-        var def = this._rpc({
-            model: 'ir.model.data',
-            method: 'xmlid_to_res_id',
-            kwargs: {xmlid: 'base.group_portal'},
-        }).then(function (id) {
-            self.group_portal_id = id;
-
-            if (self.is_manager) {
-                self.lunchUserField = self._createMany2One('users', 'res.users', self.username, function () {
-                    return [['groups_id', 'not in', [self.group_portal_id]]];
-                });
-            }
-        });
-        return Promise.all([superDef, def]);
+        if (self.is_manager) {
+            self.lunchUserField = self._createMany2One('users', 'res.users', self.username, function () {
+                return [['groups_id', 'not in', [self.group_portal_id]]];
```

Comment:
Can inline the `this` here as well IFF the `function(){}` is converted to an arrow function: `() => [['groups_id', 'not in', [self.group_portal_id]]]`.